index.html

```diff
@@ -4240,47 +4240,54 @@ <h2>HTTP</h2>
+See <a href="JSON-LD11##relationship-to-rdf">Relationship to RDF</a> for more details.
+          </p>
+          <p>
+Some scenarios where processing a <a>verifiable credential</a> or a <a>verifiable presentation</a> as RDF is
 desirable include, but are not limited to:
           </p>
-
           <ul>
             <li>
-Before securing or after verifying content
-that requires <a href="https://csrc.nist.gov/glossary/term/data_integrity">data
-integrity</a>, such as a
-<a>verifiable credential</a> or <a>verifiable presentation</a>.
+Before securing or after verifying <a>verifiable credentials</a> or <a>verifiable presentations</a>,
+in cases where the issuer has leveraged the <code>@context</code> to provide details regarding 
+their intended expression of the <a href="#verifiable-credential-graphs">Verifiable Credential Graphs</a>.
             </li>
             <li>
-When performing JSON Schema validation, as described in Section
-<a href="#data-schemas"></a>.
+After performing JSON Schema validation, as described in Section
+<a href="#data-schemas"></a>, in order to mitigate attacks 
+on the canonicalization process as described in <a href="https://w3c.github.io/rdf-canon/spec/#dataset-poisoning">Dataset Poisoning</a>
             </li>
             <li>
 When serializing or deserializing <a>verifiable credentials</a> or
-<a>verifiable presentations</a> into systems that store or index their contents.
+<a>verifiable presentations</a> into systems that store or index claims about subject in
+<code>text/turtle</code>, <code>application/n-quads</code>, <code>application/ld+json</code>. 
             </li>
             <li>
-When operating on <a>verifiable credentials</a> or <a>verifiable
-presentations</a> in a software application, after verification or validation
-is performed for securing mechanisms that require an understanding of
-and/or processing of JSON-LD.
+When validating <a>verifiable credentials</a> or <a>verifiable
+presentations</a> in a software application, after verification 
+is performed for securing mechanisms that do not require an understanding
+or processing of JSON-LD.
             </li>
             <li>
-When an application chooses to process the media type using the `+json`
+When an application chooses to process the media type using the `+ld+json`
 structured media type suffix.
             </li>
           </ul>
 
           <p>
-That is, JSON processing is allowed as long as the document being consumed or
-produced is a <a>conforming document</a>. If JSON processing is desired, an
-implementer is advised to follow the following rule:
+Naive JSON processing is allowed as long as the document being consumed or
+produced is a <a>conforming document</a>, however, this can lead to interoperability issues 
+and confusion about the specific information that has been secured between JSON-LD and RDF processors.
```

Comment:
No, "Naive JSON processing" is never allowed ... with any JSON. I guess I don't know what this means. When you are processing JSON ... or anything generally, you shouldn't be naive. There is also no special danger of confusion  or deviation of interpretation from generalized processors if the rules are followed, so we should focus there.

An attempt at rewording:

```suggestion
<a>Conforming documents</a> that are expressed using contexts and shapes that are
well-known to a consuming application can be consumed like any other JSON
document, that is, according to the rules of its associated specifications and interpreted
according to the <a>issuers</a> intent. If the <a>issuer</a> has used <code>@context</code>
to provide details regarding the expression of their <a>verifiable credential</a>, then
consumers are advised to follow these rules to ensure that the document matches
their accepted contexts and document shapes:
```

Reply:
I don't think your suggestion makes it clearer. The issuer is required to use `@context`, since it's required in conforming documents.

Perhaps you mean additional context beyond the v2 core context? can you adjust your suggestion to take that into account?

Reply:
I've made an adjustment to say "Issues might use additional contexts... and consumers are advised...".

Reply:
@OR13 please respond to this comment.

Reply:
Need to define "shapes that are well-known", assuming some version of this PR will land:

https://github.com/w3c/vc-data-model/pull/1320

I can accept this suggestion as it exists.

Reply:
@OR13,

> I can accept this suggestion as it exists.

Yes, please, thanks!